include/links.h

```diff
@@ -0,0 +1,61 @@
+#ifndef LINKS_H_
+#define LINKS_H_
```

Comment:
In Newsboat, we prefix all our include guards with `NEWSBOAT_`; please do that too.